services/addons/images/geo_msg_query/geo_msg_query.py

```diff
@@ -140,6 +149,13 @@ def run():
     )
     logging.basicConfig(format="%(levelname)s:%(message)s", level=log_level)
 
+    logging.debug("Starting the service with environment variables: ")
+    logging.debug(f"MONGO_DB_URI: {MONGO_DB_URI}")
```

Comment:
Will this expose credentials to the logs?

Reply:
Good point, I added it for debugging and forgot to remove it.